src/handbook/company/security/information-security.md

```diff
@@ -0,0 +1,195 @@
+comply with this policy.
+
+1. System level and user level passwords must comply with the [Access Control Policy]().
+
+2. Providing access to another individual, either deliberately or through failure
+to secure a device is prohibited.
+
+3. All end-user, personal (BYOD) or company owned devices used to access FlowForge
+information systems (i.e. email) must adhere to the following rules and requirements:
+
+4. Devices must be secured with a password (or equivalent control such as biometric)
+   protected screensaver or screen lock.
+
+5. Devices must not be left unattended in public.
+
+6. Users must report any suspected misuse or theft of a mobile device immediately
+   to security@flowforge.com
+
+7. Avoid sharing credentials. Secrets must be stored safely, using features such
+   as GitHub Secrets. For accounts and other sensitive data that need to be shared
+   use the company-provided password manager and ensure an appropriate scope of
+   sharing is used.
+
+9. Confidential information must not be stored on portable media such as USB drives
+
+9. Accessing FlowForge systems on public "shared" computers, such as hotel kiosks
+   is strictly prohibited
+
+10. Upon termination users agree to return all company owned devices and delete
+    all company information and accounts from any personal devices 
+
+## Acceptable Use Policy
+
+FlowForge proprietary and customer information stored on electronic and computing
+devices, whether owned or leased by FlowForge, the employee or a third party,
+remains the sole property of FlowForge for the purposes of this policy.
+
+Employees and contractors must ensure through legal or technical means that
+proprietary information is protected in accordance with the [Data Management Policy]().
+
+Google Drive should be used to store and share files within the company, ensuring
+proper access controls are applied.
+
+You have a responsibility to promptly report the theft, loss, or unauthorized
+disclosure of FlowForge proprietary information or equipment. You may access,
+use or share FlowForge proprietary information only to the extent it is authorized
+and necessary to fulfill your assigned job duties.
+
+Employees are responsible for exercising good judgment regarding the
+reasonableness of personal use of company-provided devices.
+
+For security and network maintenance purposes, authorized individuals within
+FlowForge may monitor equipment, systems and network traffic at any time.
+FlowForge reserves the right to audit networks and systems on a periodic basis
+to ensure compliance with this policy.
+
+Employees must ensure the software they use is properly licensed and used as
+intended.
+
+## Unacceptable Use
+
+Under no circumstances is an employee of FlowForge authorized to engage in any
+activity that is illegal under local, state, federal or international law while
+utilizing FlowForge-owned resources or while representing FlowForge in any capacity.
```

Comment:
I'd expected a self-reference from this document to the previously stated limitations. Can we add it?

Reply:
Not quite sure what you mean here. Which bit are you suggest we link back to?